### PR TITLE
World wide option

### PR DESCRIPTION
**Pull request by kuyucuburak**
As mentioned in https://github.com/Daniel-Ioannou/flutter_country_picker/discussions/57: I need "world wide" option in my app. @Daniel-Ioannou If you like it we can add it to the library.

# TODO
- [x] Update Country.parse method to return world wide.
- [x] Translate "World Wide" if Daniel likes it.

### CODE REVIEW

File: lib/src/country_list_bottom_sheet.dart [hunk 1]
```diff
@@ -13,6 +13,7 @@ void showCountryListBottomSheet({
   bool showPhoneCode = false,
   CountryListThemeData? countryListTheme,
   bool searchAutofocus = false,
+  bool showWorldWide = false,
```
Comment (kuyucuburak) [inline lib/src/country_list_bottom_sheet.dart:16]: Default is false for world wide option.

**Comment by kuyucuburak** [comment]
@Daniel-Ioannou I have 2 todo in this PR. I will do them if you like the idea.

**Comment by Daniel-Ioannou** [comment]
@kuyucuburak not sure how many will want this but I think we can add this option to the package. I didn't have the time to check PR code yet, please finalize your PR and I will review the PR and merge it with in the next days.

File: lib/src/res/strings/tr.dart [hunk 2]
```diff
@@ -233,14 +233,15 @@ Map<String, String> tr = {
   "UA": "Ukrayna",
   "AE": "Birleşik Arap Emirlikleri",
   "GB": "Birleşik Krallık",
-  "ABD": "Amerika Birleşik Devletleri",
```
Comment (kuyucuburak) [inline lib/src/res/strings/tr.dart:236]: As I mentioned in https://github.com/Daniel-Ioannou/flutter_country_picker/discussions/59, this had translated wrong.

**Review by kuyucuburak — COMMENTED** [review]
@Daniel-Ioannou this is ready to review! Please let me know if you have any inconvenience in the code.

**Review by Daniel-Ioannou — APPROVED** [review]

File: lib/src/res/strings/tr.dart (hunk 2 above)
Comment (Daniel-Ioannou) [reply lib/src/res/strings/tr.dart:236]: Fixed issue #63

**Pull request merged**